components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -93,5 +93,10 @@
 
     public static final String URL_SEPERATOR = "/";
     public static final String TENANT_URL_SEPERATOR = "/t/";
+
+    //Configuration for primary login identifiers
+    public static final String ENABLE_LOGIN_IDENTIFIRES = "LoginIdentiFires.Enable";
+    public static final String PRIMARY_LOGIN_IDENTIFIRE_CLAIM = "LoginIdentiFires.PrimaryLoginIdentifire";
+    public static final boolean DEFAULT_ENABLE_LOGIN_IDENTIFIRES = false;
```

Comment:
Let's fix the typos here and make sure it is reflected in the PR [1] with config additions.

[1] https://github.com/wso2/carbon-identity-framework/pull/3270